docs/json.adoc

```diff
@@ -0,0 +1,156 @@
+:page-title: JSON
+:page-description: Extends JUnit Jupiter with `@JsonFileSource`, a parametrized test that creates test based on a JSON Source
+
+The `@JsonFileSource` annotation lets you use JSON files from the classpath or local file system to create parametrized tests.
```

Comment:
I propose to slightly change the doc's structure. Add another sentence or two here what to look out for (single vs multiple arguments; nested arrays) and then add `== Method Arguments` and make the next two headers on single and multiple argument methods h3.

Reply:
Will doo that